packages/pancake-uikit/src/__tests__/components/__snapshots__/tabmenu.test.tsx.snap

```diff
@@ -0,0 +1,34 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I realise we have been doing inline snapshots, rather than generating snapshot files... but... that seems to require you to run the test, accept that it will fail, copy & paste (and tweak) the received snapshot output. 

This seems kinda inefficient, but I might be doing it wrong, and there may be a good reason for doing it like that.

Reply:
>I realise we have been doing inline snapshots, rather than generating snapshot files... but... that seems to require you to run the test, accept that it will fail, copy & paste (and tweak) the received snapshot output

I am not sure what you mean. When you run the snapshot with the `-u` parameter Jest will update the file. The idea was to avoid polluting the directory with files.

Reply:
Ah yes, of course -u would generate them automatically. 🤦 

Understood on not wanting to create lots of files - they would be neatly nestled in a _snapshots_ directory though? I'll PR a quick proposal for how we could organise our UIKit tests and see what you guys think.

Reply:
[New snapshot proposal](https://github.com/ChefHutch/pancake-toolkit/pull/1) updated and all tests moved out of this PR, so going to mark this as resolved